src/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # SPDX-FileCopyrightText: Copyright (c) 2024 Cisco Systems
 # SPDX-License-Identifier: BSD-2-Clause
 
-add_library(quicr)
+add_library(quicr SHARED)
```

Comment:
Moving to shared over static?

Reply:
Oops, this conflicts with Rich's PR to make this optional for building in Decimus